kola/tests/misc/raid.go

```diff
@@ -28,38 +28,23 @@ func init() {
 		Run:         RootOnRaid,
 		ClusterSize: 1,
 		Name:        "coreos.disk.raid.root",
-		// FIXME: This can only work on qemu, since it's overwriting
-		// /usr/share/oem/grub.cfg. The setting being appended (set
-		// linux_append="rd.auto") should probably be the default, and this
-		// should be removed after the OS level fix is made.
-		// https://github.com/coreos/bugs/issues/2099
```

Comment:
Remember to drop `Platforms`, not just the comment saying that it should be dropped.

Also add a `MinVersion`.